perses/rjmc/topology_proposal.py

```diff
@@ -881,6 +895,9 @@ def new_residue_name(self):
     @property
     def metadata(self):
         return self._metadata
+    @property
+    def core_new_to_old_atom_map(self):
```

Comment:
What is this? A dictionary of new to old, but only has the core atoms in? What does alchemical atom mean? Does it mean atoms of the old system, or atoms that are unique in the old system? If it is what I think it is then I think self._old_atoms is clearer ?